settings/dev.json

```diff
@@ -18,5 +18,6 @@
   "update_on_deploy": false,
   "include_guidance_reports": false,
   "use_production_db_config": false,
-  "copy_static_files": false
+  "copy_static_files": false,
+  "orderly_web_api_url": "http://localhost:8443/reports/api/v1"
```

Comment:
How come this is different from the suggested value for local dev? https://github.com/vimc/montagu/pull/163/files#diff-7f43a41e5e875b219434f43a6fb8437dR72

Reply:
Good question!